gapis/api/cmd_id_group.go

```diff
@@ -365,7 +365,7 @@ func (g *CmdIDGroup) AddRoot(rootidx []uint64) *SubCmdRoot {
 		return g.Spans[i].(*SubCmdRoot)
 	} else {
 		if c != 1 {
-			panic("This should not happen, a single command cannot span 2 groups")
+			panic("This should not happen, a single command cannot span %v groups", c)
```

Comment:
This does not compile:

`gapid/gapis/api/cmd_id_group.go:368: too many arguments to panic: panic("This should not happen, a single command cannot span %v groups", c)`